### PR TITLE
remove unnecessary xmlns attribute when you use a prefix without name…

### PR DESCRIPTION
**Pull request by loduis**
…spaceuri

### CODE REVIEW

File: src/xml_object.ts [hunk 1]
```diff
@@ -249,7 +249,20 @@ export class XmlObject implements IXmlSerializable {
                                 } else {
                                     node = doc.createElementNS(schema.namespaceURI, `${schema.prefix ? schema.prefix + ":" : ""}${schema.localName}`);
                                 }
-                                node!.textContent = value;
+                                if (Array.isArray(value)) {
+                                    for (let child of value) {
```
Comment (microshine) [inline src/xml_object.ts]: [tslint] Identifier 'child' is never reassigned; use 'const' instead of 'let'.

File: src/xml_object.ts [hunk 2]
```diff
@@ -249,7 +249,20 @@ export class XmlObject implements IXmlSerializable {
...
+                                        let val = child instanceof XmlObject ?
```
Comment (microshine) [inline src/xml_object.ts]: [tslint] Identifier 'val' is never reassigned; use 'const' instead of 'let'.

File: src/xml_object.ts (hunk 1 above)
Comment (loduis) [reply src/xml_object.ts]: Lint are fixed now

**Comment by microshine** [comment]
Thank you for that pull request. I published `v1.0.24`

**Pull request merged**